### ISSUE TITLE
Meeting Notes: February 24 2022 

### ISSUE THREAD

**Issue by MarsBarLee** [isabela-pf/internship#8]
# Meeting topics: 

## Follow-up on last week's to-dos
1. Audit Google Docs 8 user journeys for people using screen-readers
    - 8 User Journeys to audit
        - Opening a document from a file browser
        - Saving a document
        - Finding which user made an edit to a document in the past in the document history
        - Seeing comments
        - Responding to a comment
        - Editing a comment
        - Receiving notification of a user joining a document
        - Receiving notification of a user commenting
2. Catalog what information cannot be accessed in each user journey
    - After completing the screen-reader audit: catalog all visual-only information in Google Docs
        - For example, in Google Docs the users currently in a document are listed in the upper right via profile picture and no other way. A lot of accessibility is making sure info isn't lost, so it'll be good to know what we're accounting for.
    - After completing the keyboard audit: catalog all mouse-only information in Google Docs.
        - For example, for the Keyboard Audit, in the 'Notes' column, there is 'Can see who is on the doc (tool bar), but cannot see where they are on the doc'. Is seeing other users a re-occurring problem for keyboard accessibility?
    - We can find this information by reading the 'Notes' column in the Notion table and see what problems re-occur
3. Start making a User Journey diagram, similar to the one Swast made with her Goodreads redesign project
    - Does not need to be complete by next week.
    - Will be a useful document in the final write-u

## Updates from last week:
- 
- Celebrate the what went well!

## Discussion topics and Resources:
- Maybe discuss more on other assistive tech than screen readers

## Potential to-do items:
- Sort through [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) (check if link is correct) standards and label them as RTC problems and Independent of RTC so we can scope some accessibility areas we are not worried about

## What are you planning to work on next?
-

## Do you have any questions for someone on the team? Do you need any help from someone on the team?
-

## Scheduling next meeting
- Will anyone be out of office in the next week? Any other scheduling conflicts?

## Other
- Time for any learning/larger discussion (time permitting)
- Anything else?

**Comment by isabela-pf** [comment]
So I don't forget, I'd like to discuss starting a (very loose) draft soon! Here are some resources I'd like to use during the discussion.
- [White Paper: Purpose and Audience - Purdue Writing Lab](https://owl.purdue.edu/owl/subject_specific_writing/professional_technical_writing/white_papers/index.html)
     - I found this whole section on white papers very helpful in get concrete ideas on what needs to be covered.
- [Types of Outlines and Samples - Purdue Writing Lab](https://owl.purdue.edu/owl/general_writing/the_writing_process/developing_an_outline/types_of_outlines.html)
     - I'm just linking this page for the outlining example. I'm not really interested in it being alphanumeric, but I thought it would be helpful as an example of the level of detail we can start working with.

**Comment by MarsBarLee** [comment]
Notes to be cleaned up
Common keys- VO
Ctrl, Option: use to navigate
Unique: when someone joins document
But no update when someone has commented
Switched to voiceover, dictate everything, give more control to user
All Audit Catalogue:
inaccessible information by assistive tech
screenshot with accessible, and nonaccessible
How can visually challenged people check if they use right key? Cannot dictate which key is selected
Don't tell if person is in second paragraph

Screen-reader: read profile picture, time, comment *interesting, simplify, from reading out item to item HTML elements, not simplifity
for example, comment class that string that filled
From compliance to pleasant experience
Most of the comment isn't reactive

User journey: Swast made some user journeys
Could be a lot of possible user journeys- focus on making the 'most mileage', such as highlights different regions of UI
Or common user journeys
Thoughts in keyboard audit: how is anyone remember all these keys?

**Comment by isabela-pf** [comment]
https://youtu.be/94swlF55tVc
